std_msgs/Header header
bool gb_init
string gb_uuid
NeoMapPoint[] mps
NeoKeyFrame[] kfs